stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.27.1
+version: 1.27.2
```

Comment:
Let's bump the minor version to `1.28.0`

Reply:
OK